addons/tmf_loadouts/loadouts/cz_army_2020.hpp

```diff
@@ -424,7 +435,7 @@ class mtrtl : car {
 
 class samg : car {
 	displayName = "AA Missile Specialist";
-	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	secondaryWeapon[] = {"CUP_launch_9K32Strela"};

```

Comment:
They aren't using the Strela other than the vehicle-sized version, which is NOT the same as the manpad version. In reality they'd be using the RBS-70.

If you really want a Russian launcher, go with the Igla.